arroyo-console/src/routes/pipelines/JobOutputs.tsx

```diff
@@ -36,15 +36,17 @@ export function PipelineOutputs({ outputs }: { outputs: Array<{id: number, data:
     .reverse();
 
   return (
-    <Table maxWidth="500px">
+    <TableContainer overflow="scroll">
```

Comment:
We don't want scroll, as that creates a scroll bar even when not needed. I'd look at how the existing job preview page solves this